service/src/main/java/uk/gov/hmcts/reform/fpl/service/HearingBookingService.java

```diff
@@ -64,6 +65,10 @@
         return hearingDetails.stream().filter(this::isPastHearing).collect(toList());
     }
 
+    public List<Element<HearingBooking>> getFutureHearings(List<Element<HearingBooking>> hearingDetails) {
```

Comment:
Keep in the service or can be in case data?

Reply:
Case data 👍